engine/gamesys/src/gamesys/scripts/script_model.cpp

```diff
@@ -160,7 +160,7 @@ namespace dmGameSystem
      * ```
      */
 
-    int LuaModelComp_Play(lua_State* L)
+    int LuaModelComp_Play(lua_State* L) // Deprecated
```

Comment:
Maybe we should do something similar to what we do in script_resource for the deprecated functions:
dmLogOnceWarning("Function `resource.%s` is deprecated. Please use `liveupdate.%s` instead.", LUA_NAME, LUA_NAME); \